internal/controllers/routerinterface/tests/routerinterface-create-minimal/00-routerinterface.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Network
+metadata:
+  name: routerinterface-create-minimal
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: managed
+  resource:
+    name: routerinterface-create-minimal
```

Comment:
@mandre Incidentally, I can't remember if we wanted to be explicit with `resource.name` or not.